src/main.rs

```diff
@@ -1,17 +1,28 @@
-use std::fs;
+use std::{env, fs};
 
-use color_eyre::eyre::Result;
+use color_eyre::{
+    eyre::{Context, Result},
+    owo_colors::OwoColorize,
+};
 
 use site_builder::{Note, ZettelType};
 
 fn main() -> Result<()> {
     color_eyre::install()?;
 
-    let workdir = "/nix/persist/active-externalism/data";
-    let included_scopes = ["public"];
+    let vault_path = read_env_var("SITE_VAULT_PATH")?;
+    let output_path = read_env_var("SITE_OUTPUT_PATH")?;
+    let included_scopes = read_env_var("SITE_INCLUDE_SCOPES")?;
```

Comment:
It doesn't say anywhere that the values should be comma-separated, add that to the readme.